themes/classic/modules/ps_shoppingcart/modal.tpl

```diff
@@ -40,12 +40,12 @@
               </div>
               <div class="col-md-6">
                 <h6 class="h6 product-name">{$product.name}</h6>
-                <p>{$product.price}</p>
+                <p class="product-price">{$product.price}</p>
                 {hook h='displayProductPriceBlock' product=$product type="unit_price"}
                 {foreach from=$product.attributes item="property_value" key="property"}
-                  <span><strong>{$property}</strong>: {$property_value}</span><br>
+                  <span>{$property}: <strong>{$property_value}</strong></span><br>
```

Comment:
Adding `:` here is a bad idea because in french we have `Quelque chose :` whereas with have `Something:` in english.
Should we use translations here? @LouiseBonnard @eternoendless 
(I know this was the old behavior)

Reply:
My opinion is: if the code is written in English, then we need to use the English punctuation. It is later, during the translation, that the translators will have to respect the punctuation of the language they're translating for. Anyway, I've checked with @YeLnatSs, it is all fine now, thank you.